### ISSUE TITLE
Fix: Failing Axe accessibility tests

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2264]
`yarn test --testNamePattern="axe"`

FAIL src/components/autosuggest/_macro.spec.js
  "ARIA input fields must have an accessible name (aria-input-field-name)"
    ✕ passes jest-axe checks (498 ms)

FAIL src/components/address-input/_macro.spec.js
  "ARIA input fields must have an accessible name (aria-input-field-name)"
  "Form elements must have labels (label)"
    ✕ passes jest-axe checks (711 ms)

FAIL src/components/duration/_macro.spec.js
  "Form elements should have a visible label (label-title-only)"
    mode: multiple fields
      ✕ passes jest-axe checks (596 ms)

FAIL src/components/navigation/_macro.spec.js
  "IDs of active elements must be unique (duplicate-id-active)"
  "IDs used in ARIA and labels must be unique (duplicate-id-aria)"
    level: container
      ✕ passes jest-axe checks (919 ms)

FAIL src/components/tabs/_macro.spec.js
  "ARIA role should be appropriate for the element (aria-allowed-role)"
    ✕ passes jest-axe checks (231 ms)
